lib/src/node/value/calculation.dart

```diff
@@ -62,11 +53,14 @@ final JSClass calculationClass = () {
       return SassCalculation.unsimplified('max', argList);
     },
     'clamp': (Object min, [Object? value, Object? max]) {
-      if (value == null) {
-        assertUnquotedStringOrInterpolation(min);
-      } else if (max == null) {
-        assertUnquotedStringOrInterpolation(min);
-        assertUnquotedStringOrInterpolation(value);
+      if ((value == null && !isValidClampArg(min)) ||
+          (max == null) && !([min, value]).any(isValidClampArg)) {
```

Comment:
I think the parens are slightly wrong here:

```suggestion
          (max == null && ![min, value].any(isValidClampArg))) {
```